### DISCUSSION TITLE
ReplayGain and no stuttering audio when switching apps.

### DISCUSSION THREAD

**Discussion by Athzzzx** [FossifyOrg/General-Discussion#578] [Ideas]
Please add ReplayGain and make the audio not stutter when switch apps.

**Comment by naveensingh** [comment]
> ReplayGain

Checkout https://github.com/FossifyOrg/Music-Player/issues/88

> make the audio not stutter when switch apps.

Not sure what you mean, can you share more information or steps to reproduce?

**Comment by naveensingh** [comment]
Keeping it open for visibility.

**Reply by Athzzzx** [reply]
Sorry for late reply.

What I meant is that when switching between apps, the playing audio stutter/stop for less than a second then resume playing. The native music player doesn't have this issue. Another foss app doesn't have this is Symphony, while this app and Auxio has it.

Sorry for the long description I don't know how to reproduce this issue here 😔.

**Reply by naveensingh** [reply]
Oh that makes sense now. What device do you have?

It's closed as a device issue in Auxio: https://github.com/OxygenCobalt/Auxio/issues/872

**Reply by Athzzzx** [reply]
> Oh that makes sense now. What device do you have?
> 
> It's closed as a device issue in Auxio: https://github.com/OxygenCobalt/Auxio/issues/872 

I'm using Xperia 1 III.

If I remember correctly, from the apps I tried only Symphony doesn't stutter, while Auxio, this app, VinylMusicPlayer, etc. has it.

**Reply by naveensingh** [reply]
It probably is a device thing but I have created a new issue for this: https://github.com/FossifyOrg/Music-Player/issues/123

**Reply by Athzzzx** [reply]
> It probably is a device thing but I have created a new issue for this: https://github.com/FossifyOrg/Music-Player/issues/123

Thank you very much
